### PR TITLE
Make leukocyte bionic toggled

### PR DESCRIPTION
**Pull request by Coolthulhu**
While active, `healthy_mod` will effectively be 100. This is way slower than its `master` incarnation which can grant 100 health in a nanosecond. You can expect to max it out after 300 health checks, which translates to 75 days. On the other hand, it protects from unhealthy effects quite well, pretty much making them not count.
That 75 days is an issue of the whole health system having very slow reactions (only 4 health checks per day). The bionic is still the fastest way of achieving 70+ health - and the only feasible one.

Also added a minor side effect, more for flavor than actual penalties:
Active leukocyte breeder will completely destroy `healthy_mod`. This won't have an effect until the bionic is turned off, then it will speed up degradation of health. Not much, because `healthy_mod` balances way too quickly.
This represents dependency on the bionic.

### CODE REVIEW

File: data/json/bionics.json [hunk 1]
```diff
@@ -536,8 +536,11 @@
         "occupied_bodyparts": [
             [ "TORSO", 10 ]
         ],
-        "act_cost": 50,
-        "description": "You are equipped with bionic stimulators which augment your haematopoiesis system, allowing you to accelerate white blood cell production using bionic power."
+        "toggled": true,
+        "act_cost": 10,
+        "react_cost": 1,
+        "time": 600,
+        "description": "You are equipped with bionic stimulators which augment your haematopoiesis system, allowing you to accelerate white blood cell production using bionic power.  It is supposed to run continously and may cause unpleasant side effects when turned off."
```
Comment (illi-kun) [inline data/json/bionics.json]: Typo: ```continously ``` -> ``` continuously ```

File: data/json/bionics.json [hunk 2]
```diff
@@ -536,8 +536,11 @@
         "occupied_bodyparts": [
             [ "TORSO", 10 ]
         ],
-        "act_cost": 50,
-        "description": "You are equipped with bionic stimulators which augment your haematopoiesis system, allowing you to accelerate white blood cell production using bionic power."
+        "toggled": true,
+        "act_cost": 10,
+        "react_cost": 1,
+        "time": 600,
```
Comment (illi-kun) [inline data/json/bionics.json]: 1PU/600 turns is nearly free, do you think that's fine balanced?

Comment (Coolthulhu) [inline data/json/bionics.json]: Tried to keep it relatively similar to what it is in master and avoid exploits.

The problem is, health checks happen only once per 6 hours (3600 turns). It shouldn't be possible to activate the bionic just for the check to save power. So activation cost should be no less than the cost of keeping it active for 6 hours.

Then finally, bypassing the health system doesn't really mean that much. If the CBM limit was turned on right now, leukocyte wouldn't be worth it, except for those who get irradiated all the time.

Comment (illi-kun) [inline data/json/bionics.json]: This explanation makes sense, so only issue preventing this PR from merge is a typo mentioned above.

**Pull request merged**